simplecpp.cpp

```diff
@@ -1927,7 +1932,7 @@ namespace simplecpp {
                 }
             }
 
-            Token * const output_end_1 = output.back();
+            const Token * const output_end_1 = output.back();
```

Comment:
This is not semantically const. Maybe a NOLINT would be more appropriate.

Reply:
Done.